scripts/package_chrome_extension.sh

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+PREVIOUS_CURRENT_DIR=$(pwd)
+VERSION=$1
+cd ..
+echo "Packaging unsigned Chrome extension, version $VERSION"
+zip -r build/kiwix-chrome-unsigned-extension-$VERSION.zip www webextension manifest.json LICENSE-GPLv3.txt service-worker.js README.md
+# Package the extension with Chromium
+# On Travis, we need to make it believe it has a display, because --headless only works on Chromium>=57, which is not available on Travis for now
+if [ ! -z ${TRAVIS_BUILD_DIR+x} ]; then
+    export DISPLAY=:99.0
+    sh -e /etc/init.d/xvfb start
+fi
```

Comment:
This could probably be made by travis itself in `.travis.yml`.

Btw, there is a lot of travis specific setting. Setting all of them in `.travis.yml` could be a bit annoying and difficult to maintain. Maybe you could move all travis specific setting in a script `setup_travis_env.sh`